PWGGA/GammaConv/AliAnalysisTaskGammaCaloMergedML.h

```diff
@@ -324,21 +323,23 @@ class AliAnalysisTaskGammaCaloMergedML : public AliAnalysisTaskSE {
 
 
    // Trees for Machine learning
-
-    TTree**                 tTrueMergedCaloClusterPi0;                          //! Trees with MC information of merged clusters from pi0
-    TTree**                 tTrueMergedCaloClusterEta;
-    TTree**                 tTrueMergedCaloClusterBck;                          //! Tress with MC information of merged cluster from background
-
-  //Float_t**               fMergedClusterTreeCluster;
-    Float_t                 fMergedClusterTreeCluster[50][50];
-    Int_t                   fMergedClusterTreeClusterType;
-    Float_t                 fMergedClusterTreeEnergy;
-    Float_t                 fMergedClusterTreePt;
-    Int_t                   fMergedClusterTreeModNum;
-    Int_t                   fMergedClusterTreeXPos;
-    Int_t                   fMergedClusterTreeYPos;
-    Int_t                   fMergedClusterTreeIsPrimary;
-    Int_t                   fMergedClusterTreePID;
+    TTree**                 tTrueMergedCaloClusterPi0;                          //! array of trees with MC information of merged clusters from pi0
+    TTree**                 tTrueMergedCaloClusterEta;			   //! array of trees with MC information of merged clusters from eta
+    TTree**                 tTrueMergedCaloClusterBck;                          //! array of tress with MC information of merged clusters from background
+
+    Float_t                 fMergedClusterTreeCluster[50][50];		   //! energy of cluster
+    Int_t                   fMergedClusterTreeClusterType;			   //! cluster classification
```

Comment:
be aware of data types. Int_t is very big. I recommend wherever possible to use Short_t or UShort_t